ui-components/autocompleter.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Autocompleter
+title_nav: Autocompleter
+description: Add a custom autocompleter to TinyMCE 5.0.
+keywords: autcomplete
+---
+
+## Overview and use case
+
+An `autocompleter` enables users to quickly find and select from a pre-populated list of values as they type, leveraging searching and filtering. It provides suggestions to insert while the user is typing into the content. For example, typing **:amp** should show the ampersand item in the menu. Pressing `esc` should close the autocomplete menu.
+
+## How to create custom autocompleters
+
+The method for adding a custom autocompleter is in the **UI Registry** part of the editor API `editor.ui.registry`:
+
+* `editor.ui.registry.addAutocompleter(name, configuration)`
+
+The two arguments this method take are:
+
+* `name` - A unique name for the autocompleter.
+* `configuration` - An object containing the user's configuration.
+
+### Configuration options
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| ch | string | Required | The character to trigger the autocompleter. |
```

Comment:
We should really rename this to something more descriptive in the code. Now would be a good time, before we merge these docs. @TheSpyder @EkimChau thoughts?